Data/Aeson/Types/Generic.hs

```diff
@@ -201,7 +201,6 @@ instance ( IsRecord      a isRecord
         (builder tagFieldName <>
          B.char7 ':' <>
          builder (constructorTagModifier opts (conName (undefined :: t c a p)))) <>
-        B.char7 ',' <>
```

Comment:
I think I see what's going on: you moved the comma insertion elsewhere. Is that correct?

(Ideally, the movement of comma insertion would have been a separate standalone commit, and it would thus have been obvious what was going on.)